mono/metadata/assembly.c

```diff
@@ -133,36 +135,140 @@ static const AssemblyVersionMap framework_assemblies [] = {
 	{"Novell.Directory.Ldap", 0},
 	{"PEAPI", 0},
 	{"System", 0},
+	{"System.AppContext", 0},
```

Comment:
Because https://github.com/lambdageek/mono/blob/2cd2eef5de3217bc4f2365bdb51a85b03132abc1/mono/metadata/assembly.c#L3512 hack does not always work (xammac issue), @lambdageek could we remove it now